Gemfile

```diff
@@ -43,9 +43,11 @@ group :system_tests do
 
   gem 'beaker-rspec'
   gem 'beaker-windows'
-  gem 'net-ssh'
+  # Need at least 85ebe31 for EL9 support
+  gem 'net-ssh', :git => "https://github.com/net-ssh/net-ssh", :ref => 'v7.0.0.beta1'
+  gem 'bcrypt_pbkdf' unless RUBY_PLATFORM == 'java'
   gem 'puppet', ENV.fetch('PUPPET_VERSION', '~> 7.0')
+  gem 'puppetlabs_spec_helper', '~> 3.0'
```

Comment:
This looks to be reverting the bump to version 4

Reply:
d'oh! Had to rebase :-/